src/routes/docs/tutorials/react/step-6/+page.markdoc

```diff
@@ -50,7 +50,7 @@ export function IdeasProvider(props) {
       ID.unique(),
       idea
     );
-    setIdeas((ideas) => [response.$id, ...ideas].slice(0, 10));
```

Comment:
What's this change?

Reply:
Found a mistake in the original react tutorial. The response from createDocument is a single document. The ideas array is an array of documents